tools/rosbag/src/player.cpp

```diff
@@ -201,7 +202,7 @@ void Player::publish() {
     options_.advertise_sleep.sleep();
     std::cout << " done." << std::endl;
 
-    std::cout << std::endl << "Hit space to toggle paused, or 's' to step." << std::endl;
+    std::cout << std::endl << "Hit space to toggle paused, or 's' to step, [ or ] to decrease/increase playback speed, backspace to set speed to normal." << std::endl;
```

Comment:
`Hit space to toggle paused, 's' to step, '['/']' to decrease/increase playback speed, or backspace to set speed to normal.`